apps/project/models.py

```diff
@@ -304,10 +311,19 @@ def clean(self):
         #     self.requiredResults += group.requiredCount * group.numberOfTasks
 
 
+class SoftDeletionManager(models.Manager):
+    @typing.override
+    def get_queryset(self):
+        return super().get_queryset().filter(is_deleted=False)
```

Comment:
Let's not use this.... client should handle this

Add a validation in the project serializer... (Or unset is_deleted if used)